website/content/partials/faq/client-count/computing-clients.mdx

```diff
@@ -88,3 +88,27 @@ If you are using Vault 1.10+:
 
 If you are using an older version of Vault, migrating mounts will always create 
 duplication in the client count.
+
+### Can I take parts of the full payload and add them up to get the "total clients"? ((#adding-clients))
+
+Probably not.
```

Comment:
I think that this line might be confusing, and might make the reader doubt the paragraphs below. I'd advocate for removing it (but keeping all of the other paragraphs of this FAQ answer).